source/stylesheets/screen.css.scss

```diff
@@ -91,3 +91,11 @@ ul.subdir__menu {
 .indented-list {
   margin: 0 0 0 20px;
 }
+
+.architecture-diagram {
+  height: 700px;
+  width: calc(100vw - 400px);
+  @include govuk-media-query($until: tablet) {
+    width: calc(100vw - 60px);
```

Comment:
Hello 👋 I wonder if this would be clearer with some variables `calc(100vw - (#{$govuk-gutter} * 2))`

I haven't tested this ☝️, but it should work

Reply:
Ah yeah, I was thinking of doing that but I was too lazy to look up the variable name 😅 

I'll give it a try, thanks!